src/utils/GovNotifyEvent.ts

```diff
@@ -0,0 +1,30 @@
+import { PersonIdentity } from "../models/PersonIdentity";
+import { personIdentityUtils } from "./PersonIdentityUtils";
+
+export type GovNotifyName =
+	"F2F_CRI_START"
+	| "F2F_CRI_AUTH_CODE_ISSUED";
```

Comment:
We dont need the second name F2F_CRI_AUTH_CODE_ISSUED?

Reply:
This needs to be replaced just needed to know what event name to replace with: https://github.com/alphagov/di-ipv-cri-f2f-api/pull/37#discussion_r1157596900 but if your changes aren't expecting a specific event_name I'll make something up :)